_posts/2022-08-03-WordPress-common-troubleshooting-scenarios.md

```diff
@@ -0,0 +1,85 @@
+ 4. If you're still facing mixed content errors, search for HTTP references in the database and replace them with HTTPS. You can easily do it using the plugins below -
+[Better Search and Replace](https://wordpress.org/plugins/better-search-replace/)
+[Search & Replace](https://wordpress.org/plugins/search-and-replace/)
+
+5. Please install the WordPress plugin 'SSL Insecure Content Fixer' and select the appropriate HTTPS detection method as in the screenshot below. For example,HTTP_X_ARR_SSL setting works for Azure Linux App Service blessed images. 
+ >![WordPress common troubleshooting scenarios](/media/2022/08/wp_detect_http_setting.png)
+
+# How to Debug WordPress Errors
+If your WordPress page is encountering application issues without any detailed errors, please use the below steps to debug further. 
+  >![WordPress common troubleshooting scenarios](/media/2022/08/wp_critical_error.png)
+
+1.	Edit the wp-config.php file and look for the following line of code:
+<br/>define(‘WP_DEBUG’, false);
+<br/>Please update the line to the changes below - 
+<br/>define(‘WP_DEBUG’, true);
+<br/>define(‘WP_DEBUG_LOG’, true);
+<br/>define(‘WP_DEBUG_DISPLAY’, false);
+<br/>@ini_set(‘display_errors’,0);
+2.	Check debug.log inside the wp-content folder and application logs for the detailed errors. 
+3.	Once you fix the issue please revert the changes done in step 1.
+
+# How to resolve the 'Too Many Redirects' issue encountered on your WordPress site
+1.	 Assess if the SSL, home_url and site_url configurations are set properly in your wp-config.php. Please refer to the screenshot and code below for the recommended settings.
+<br/>if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https')
+<br/>$_SERVER['HTTPS'] = 'on';
+<br/>$http_protocol='http://';
+<br/>if (!preg_match("/^localhost(:[0-9])*/", $_SERVER['HTTP_HOST']) && !preg_match("/^127\.0\.0\.1(:[0-9])*/", $_SERVER['HTTP_HOST'])) {
+<br/>	$http_protocol='https://';
+<br/>}
+<br/>define('WP_HOME', $http_protocol . $_SERVER['HTTP_HOST']);
+<br/>define('WP_SITEURL', $http_protocol . $_SERVER['HTTP_HOST']);
+ >![WordPress common troubleshooting scenarios](/media/2022/08/wp_home_site_url.png)
+2.	Disable all plugins by renaming the plugins folder to check if the redirection is being caused by any of the plugins. You can then re-enable them one by one to find the problematic plugin.
+3.	Please check if the WordPress admin URL https://<site home url>/wp-admin or any other URLs within the admin page are also facing the 'too many redirects' issue. If only a few URLs are redirecting it could be a incorrect rediect rule configured in the Redirect Plugins. Also, review the server configuration files like Nginx config files, .htaccess (in case you are using an Apache Web Server)  or web.config (in case you are using the IIS Web Server) for any redirect rules that may have been configured and modify/update the configurations as needed. 
```

Comment:
```suggestion
3.	Please check if the WordPress admin URL https://<site home url>/wp-admin or any other URLs within the admin page are also facing the 'too many redirects' issue. If only a few URLs are redirecting it could be an incorrect redirect rule configured in the Redirect Plugins. Also, review the server configuration files like Nginx config files, .htaccess (in case you are using an Apache Web Server)  or web. config (in case you are using the IIS Web Server) for any redirect rules that may have been configured and modify/update the configurations as needed. 
```